app/server/src/features/accounts/methods.ts

```diff
@@ -209,4 +211,57 @@ export async function deleteAccount(prisma: PrismaClient, input: DeleteAccountFo
 
     // return deleted user
     return { deletedUser };
-}
\ No newline at end of file
+}
+
+export async function resetPasswordRequest(prisma: PrismaClient, input: ResetPasswordRequestForm) {
+    const result = await prisma.user.findUnique({
+        where: { email: input.email }
+    });
+
+    const accountFound = !!result;
+
+    // No need to send email if the account does not exist
+    if (!accountFound) throw new Error('No account with that email was found');
+
+    const token = await jwt.sign({ email: input.email });
+    const passwordResetLink = `prytaneum.io/reset-password?token=${token}`
+
+    try {
+        await sendEmail({
+            to: input.email,
+            subject: 'Password Reset',
+            template: 'password-reset',
+            'h:X-Mailgun-Variables': JSON.stringify({ passwordResetLink }),
+        });
+        return accountFound;
+    } catch (err) {
+        server.log.error(err);
+        throw new Error(errors.email)
```

Comment:
Does this error get sent to the client?  I can't remember if we mask server errors from being sent to the client.  I think we do.